src/config/index.ts

```diff
@@ -55,4 +55,8 @@ export const env = readEnv(process.env, {
                 'OFF is intended to be used to turn off logging, not as a level for actual logging.'
         )
         .default('info'),
+    CALLBACK_URL: str.describe('Address to resend validated file Id after validating with account validation web')
+        .default('http://chs.local/account-filing/uploaded/{fileId}'),
+    SUBMIT_VALIDATION_URL: str.describe('Url address to upload file to be validated with account validation web')
+        .default('http://chs.local/xbrl_validate/submit'),
```

Comment:
I don't think these values should have deaults as if they are not configured the service will not work so it doesn't make sense to have a default.